### PR TITLE
Better Readme for Github

### PR DESCRIPTION
**Pull request by whyisjake**
Let's make something that looks a little better on Github.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+### 1.1.1 (Apr. 15, 2009) 
+* Changed SQL query to return only contributor-level and above users.
+
+### 1.1.0 (Apr. 14, 2009) 
+* Initial beta release.
+
+## Installation 
+
+1. IMPORTANT: Please disable the original Co-Authors plugin (if you are using it) before installing Co-Authors Plus
```
Comment (bcampeau) [inline readme.md]: Numbering seems off here?

Comment (whyisjake) [inline readme.md]: What do you mean?

Comment (bcampeau) [inline readme.md]: Odd, diff is showing this numbered as:
1.
1.
1.
1.
1.

Looks fine on viewing though!

Comment (whyisjake) [inline readme.md]: Yeah, Markdown doesn't care. :)

**Pull request closed without merging**